spec/amp-cache-debugging.md

```diff
@@ -0,0 +1,23 @@
+# AMP Cache Debugging
+
+**Why is my doc broken on an AMP cache?**
+
+Valid AMP documents typically appear and behave the same on AMP Caches as they
+do on the origin. However, there are some components and server configurations
+that can be problematic.
+
+If a particular document appears and behaves as expected on your origin, but not
+when viewed via the cache (how to map origin URLs to [Google's AMP Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)),
```

Comment:
Not at the moment.  We have in our backlog.  

We are actually following the same format.  I am not sure if there is comparable doc on am-project.org.